bundler.go

```diff
@@ -138,6 +141,7 @@ type Bundler struct {
 	pathWorkingDirectory string
 	pathManifest         string
 	resourcesAdapters    []ConfigurationResourcesAdapter
+	KeepWindowsConsole   bool
```

Comment:
Could you make the first letter a lower case letter (`keepWindowsConsole`) so that it's not exported?